components/organisms/Header.js

```diff
@@ -1,34 +1,12 @@
 import PropTypes from "prop-types";
 
-import { SearchBar } from "../atoms/SearchBar";
-import { useState } from "react";
-
 export function Header(props) {
-  //State for Search input
-  const [searchText, setSearchText] = useState(null);
-
-  //Handler for search input
-  function handleInputChange(e) {
-    setSearchText(e.target.value);
-  }
-
   return (
-    <div className="w-full flex  h-auto flex-col mt-8 lg:flex lg:flex-row  xl:flex xl:flex-row   justify-between ">
```

Comment:
why was the search bar removed from here. It doesn't make sense to have a component just for an image

Reply:
I moved the search bar here because the link for the translation would not work if it went on the header component. It would give me fs is null. I moved the search bar out so that I can match it with the design.